src/AiidaDFTK.jl

```diff
@@ -101,7 +102,36 @@ Run a DFTK calculation from a json input file.
 Output is by default written to `stdout` and `stderr`.
 The list of generated output files is returned.
 """
-function run_json(filename::AbstractString; extra_output_files=String[])
+function run_json(filename::AbstractString; extra_output_files=String[], min_version=typemin(VersionNumber), max_version=typemax(VersionNumber))
+    # We write a few key messages to an error file that we can parse from
+    # the AiiDA plugin to detect common failure modes.
+    errorfile = "errors.log"
+    if isfile(errorfile)
+        error("Error file $errorfile already exists!")
+    end
+    push!(extra_output_files, errorfile)
+    function report_error(msg)
```

Comment:
I would prefer free functions here (pass `errorfile` each time or better make it a package-global constant). Actually it also feels like you are re-doing the `Logging` package. Why not setup something using that infrastructure ?

Reply:
We can reuse some of the logging infrastructure, however I fear that this will increase complexity for no benefit. (We'd have to tag messages with a specific `group` and implement a custom logger to only print a few select messages...) 😅

Reply:
Let's try it anyway. Maybe we reintroduce the logging capture after all...

Reply:
OK I switched it to the logging infrastructure, it works quite well if one is careful about the details. (e.g. `SimpleLogger` does not flush automatically, but `FileLoggers` from `LoggingExtras` does).